datalad/interface/rerun.py

```diff
@@ -491,6 +492,19 @@ def get_run_info(message):
             'cannot rerun command, command specification is not valid JSON: '
             '%s' % exc_str(e)
         )
+    if not isinstance(runinfo, (list, dict)):
+        # this is a run record ID -> load the beast
+        record_dir = dset.config.get(
+            'datalad.run.record-directory',
+            default=op.join('.datalad', 'runinfo'))
+        record_path = op.join(dset.path, record_dir, runinfo)
+        if not op.lexists(record_path):
+            # too harsh IMHO, but same harshness as few lines further down
```

Comment:
The ValueError is caught and eventually converted to a result with an error status.  Do you think it should be "impossible" and skipped instead?

Reply:
Ah, that sounds good. No, I think it is an error, I just wouldn't want it to crash. But if it yields an 'error' result eventually I am happy.